### ISSUE TITLE
[feature_request] Organise photos in Polarr (projects)

### ISSUE THREAD

**Issue by skrimaging** [Polarrco/polarr_mac_support#29]
Right now I can import a bunch of Raw files directly into Polarr from my Mac Finder.

Once in Polarr, i have no way to organize the imported files into folders or projects.

I would love to be able to create a project and then drag photos to it. That way when I reopen Polarr, I can select which project (photoshoot) I want to work on instead of always having to reimport. 

This way, the edits I make could be saved and I could come back to an edited photo and change something without having to re edit from scratch.

**Comment by kopaka7** [comment]
Yeah, I agree. I've found the 50 photo limit for imports to be a bit limiting because there's no way to save the file edits beyond exporting. I guess the Photos extension sort of lets you use projects, but that is more limiting because it only saves the original file and the edited file, not the edits made in between. @skrimaging You know you can import up to 50 photos and save their edit history, though, right?

**Comment by kpaddie** [comment]
Hi there,

We have, in the past, considered something of this nature. Storage and album management are not currently in the core functionality of the app. Primarily, Polarr is an editing tool, and as you probably know, it never stores your photos, and the 50 photo limit is a result of the caching process. That being said, we are always expanding and improving the app to make it better, so if this becomes something our users overwhelmingly want, it is something we may consider.

Very best,

-Polarr team

**Comment by skrimaging** [comment]
Hi @kpaddie

Is it possible to  reopen the topic? That way users could chime in their thoughts.. It is the reason I started the topic to see if others were also interested.

**Comment by kpaddie** [comment]
Certainly. The issue has been reopened.

**Comment by kopaka7** [comment]
I'm glad to see the 50 photo limit has been increased. I noticed by accident recently when I imported extra pictures and crossed the 50 mark. What is the new limit?

**Comment by seanpkent** [comment]
@kopaka7 The new import limit is 200

**Comment by skrimaging** [comment]
awesome.. this new 200 limit is great

**Comment by Martasdx** [comment]
There's still 50 photos limit in Mac app. Can you increase it as well?
